packages/autorest.go/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 4.0.0-preview.72 (unreleased)
```

Comment:
Could you set this to today's date? I'm going to release this emitter per request of @JiaqiZhang-Dev